Rakefile

```diff
@@ -4,16 +4,38 @@
 
 protoc_binary="protoc"
 protoc_version="3.5.1"
-gogo_dir="/tmp/gogo"
+gogo_dir=Dir.pwd + "/toolchains/gogo"
+gogo_bin=Dir.pwd + "/toolchains/gogo-bin-v1.3.2"
```

Comment:
I haven't worked w/ Ruby in a long time, but I think this is safer:
```ruby
gogo_dir=File.join(File.dirname(__FILE__), "toolchains", "gogo")
gogo_bin=File.join(File.dirname(__FILE__), "toolchains", "gogo-bin-v1.3.2")
```

Separately, could we extract the exact version into it's own variable that gets reused below for the tag checkout?